Snowplow/SPTracker.m

```diff
@@ -247,6 +248,48 @@ - (void) setSessionContext:(BOOL)sessionContext {
     }
 }
 
+- (void) setGDPRContextWithBasis:(SNOWProcessingBasis)basisForProcessing
+         withDocumentDescription:(NSString *)documentDescription
+             withDocumentVersion:(NSString *)documentVersion
+                  withDocumentId:(NSString *)documentId {
+    SPPayload * pl = [[SPPayload alloc] init];
+    [pl addValueToPayload:documentId forKey:@"documentId"];
+    [pl addValueToPayload:documentVersion forKey:@"documentVersion"];
+    [pl addValueToPayload:documentDescription forKey:@"documentDescription"];
```

Comment:
The `kSPDocumentId` etc should be used for the keys here to follow the prior pattern - you will need to define them in Snowplow.m first however before you can use them!